server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/MailboxGetMethod.scala

```diff
@@ -43,7 +43,7 @@ import reactor.core.scheduler.Schedulers
 
 import scala.jdk.CollectionConverters._
 
-class MailboxGetMethod @Inject() (serializer: Serializer,
+class MailboxGetMethod @Inject() (serializer: MailboxSerializer,
```

Comment:
Same remark

Reply:
Idem, its not an object as there is a dependency to the MailboxId.Factory